Kudu.Contracts/Deployment/DeploymentInfoBase.cs

```diff
@@ -1,8 +1,8 @@
 ﻿using System;
-using Kudu.Core.SourceControl;
 using Kudu.Contracts.Tracing;
 using System.Threading.Tasks;
 using Kudu.Contracts.Deployment;
+using Kudu.Core.SourceControl;
```

Comment:
Do we need this ?

Reply:
Yes it is used by several class types in the document.